modules/nf-core/ensemblvep/vep/main.nf

```diff
@@ -47,6 +48,9 @@ process ENSEMBLVEP_VEP {
         --dir_cache $dir_cache \\
         --fork $task.cpus
 
+    if [ $file_extension == vcf ]; then
+        tabix ${prefix}.${file_extension}.gz
+    fi
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
```

Comment:
Tabix version check? Perhaps also push this upstream?

Reply:
good idea!